### PR TITLE
docs: updated error i18n messages

### PR DESCRIPTION
**Pull request by JessieWuJiexi**
#### What type of PR is this?

type-docs

#### What this PR does / why we need it:

updated error messages to make sure they are readable

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-core/src/main/resources/i18n/ErrorMessages.properties [hunk 1]
```diff
@@ -4,174 +4,174 @@
...
+com.oceanbase.odc.ErrorCodes.DebugObjectUnsupported=OBServer does not support debugging the current PL object.
+com.oceanbase.odc.ErrorCodes.DebugSessionNotAlive=The debugging session has expired. Details: {0}.
+com.oceanbase.odc.ErrorCodes.DebugStartFailed=Failed to start debugging. Cause: {0}.
+com.oceanbase.odc.ErrorCodes.DebugInfoParseFailed=Failed to parse the debugging information. Please check if the database version is the latest.
+com.oceanbase.odc.ErrorCodes.DebugTimeout=Debugging timed out. Please close the window and try again.
+com.oceanbase.odc.ErrorCodes.PLObjectFetchFailed=Fail to fetch the DDL of the PL object. Details: {0}.
+com.oceanbase.odc.ErrorCodes.MockError=An error occurred for the mock data ticket. Details: {0}.
+com.oceanbase.odc.ErrorCodes.ImportInvalidZip=Invalid ZIP file. Details: The ZIP file is not exported by ODC or OBDUMPER.
+com.oceanbase.odc.ErrorCodes.ImportInvalidFileType=Invalid file type {0}. Details: Import is not supported for this file type.
+com.oceanbase.odc.ErrorCodes.FlowTaskInstanceExpired=The execution of the workflow timed out.
```
Comment (yizhouxw) [inline server/odc-core/src/main/resources/i18n/ErrorMessages.properties:97]: user may confused for 3 differenct concept: workflow, ticket and task.
we may unify usage, such as
- while describe as an approve workflow, use 'ticket'
- whiel descrive the job execution, use 'task'

**Review by yizhouxw — COMMENTED** [review]
great job 👍 

several comment below, let's have a discussion if possible.

**Review by yizhouxw — APPROVED** [review]
👍👍 professional !

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by Jane201510 — COMMENTED** [review]
nice.

**Review by Jane201510 — APPROVED** [review]

**Pull request merged**